lisa/operating_system.py

```diff
@@ -191,19 +336,65 @@ class Linux(Posix):
 
 
 class Debian(Linux):
+    __lsb_os_info_pattern = re.compile(
+        r"^(?P<name>.*):(\s+)(?P<value>.*?)?$", re.MULTILINE
+    )
+
     @classmethod
     def name_pattern(cls) -> Pattern[str]:
         return re.compile("^debian|Forcepoint|Kali$")
 
     def _initialize_package_installation(self) -> None:
         self._node.execute("apt-get update", sudo=True)
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> None:
         command = (
             f"DEBIAN_FRONTEND=noninteractive "
             f"apt-get -y install {' '.join(packages)}"
         )
-        self._node.execute(command, sudo=True)
+        if not signed:
+            command += " --allow-unauthenticated"
+
+        install_result = self._node.execute(command, sudo=True)
+        if install_result.exit_code != 0:
+            raise LisaException(
+                f"Failed to install {packages}." f" stdout: {install_result.stdout}"
```

Comment:
print out `exit_code` and make sure the `stdout` is meaningful, and no flood information.

Reply:
Ok, is this the only thing in current iteration?

Reply:
Yes, I have a look on current content. Let's see if others have more comments. If there is no, I will approve and merge it.